data/data/bootstrap/baremetal/files/etc/NetworkManager/system-connections/ens3.nmconnection.template

```diff
@@ -0,0 +1,15 @@
+{{if .PlatformData.BareMetal.ExternalStaticIP}}
+[connection]
+type=ethernet
+interface-name=ens3
```

Comment:
@zaneb can confirm. ARM and x86 libvirt vms will end up with different nic names because they use different pci/pcie busses by default.

Reply:
Since #5698 switched to referring to the nics by MAC (either user specified or generated) I think we can just modify the NM keyfile to refer to the mac instead of the device name?